.template/addons/github/.github/workflows/publish_wiki.yml.tt

```diff
@@ -2,17 +2,35 @@ name: Publish Wiki
 
 on:
   push:
-    paths:
-      - .github/wiki/**
     branches:
       - develop
+    paths:
+      - .github/wiki/**
+
+env:
+  USER_NAME: github-wiki-workflow
+  USER_EMAIL: bot@nimblehq.co
 
 jobs:
   publish:
-    name: Publish Wiki
-    uses: nimblehq/github-actions-workflows/.github/workflows/publish_wiki.yml@0.1.0
-    with:
-      USER_NAME: github-wiki-workflow
-      USER_EMAIL: ${{ secrets.GH_EMAIL }}
-    secrets:
-      USER_TOKEN: ${{ secrets.GH_TOKEN }}
+    name: Publish Github Wiki
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Publish Github Wiki
+        uses: nimblehq/publish-github-wiki-action@v1.0
+        with:
+          user_name: ${{ env.USER_NAME }}
+          user_email: ${{ env.USER_EMAIL }}
```

Comment:
Why don't we hardcode it,  this `env` is used only once 💭 

```suggestion
          user_name: github-wiki-workflow
          user_email: bot@nimblehq.co
```

Reply:
Thanks 👍🏼 . Fixed in f68814abcb4b2b56cbc8a26fea64e0b865337766